nmmo/systems/combat.py

```diff
@@ -92,9 +92,9 @@ def attack(realm, player, target, skillFn):
     damage  = max(int(damage), 0)
 
     if player.is_player:
-        realm.log_milestone(f'Damage_{skill_name}', damage,
+        realm.log_milestone(f'[PlayerID: {player.ent_id}] Damage_{skill_name}', damage,
```

Comment:
Again, I feel like the first arg to log_milestone should be a milestone name. It looks like you might want to add tags={"PlayerID": player.ent_id} which seems like a good idea

Reply:
Just added tags to realm.py, Realm.log_milestone(), and used it through out!